dependencies/secret_service.go

```diff
@@ -0,0 +1,10 @@
+package dependencies
+
+import (
+	"context"
+)
+
+type SecretService interface {
```

Comment:
Can this get a godoc comment?

Reply:
I don't think this needs to specifically mention dependency injection. I think it just needs to say "it is a service for looking up a secret based on the key."